packages/app-project/src/shared/components/SubjectPicker/locales/en.json

```diff
@@ -1,6 +1,9 @@
 {
   "SubjectPicker": {
     "heading": "Choose a subject to get started",
-    "byline": "Sort the list by clicking column names. You will see subjects sequentially, starting with the one you choose."
+    "byline": "Sort the list by clicking column names. You will see subjects sequentially, starting with the one you choose.",
+    "alreadySeen": "Already seen",
+    "retired": "Retired",
+    "unclassified": "Unclassified"
```

Comment:
@snblickhan should this say 'Unseen'?

Reply:
Or 'Available', after conversation on Slack.

Reply:
I've made that wording change, so this is ready now.

Reply:
Thanks! Yes, can confirm we'd settled on "Available" "Already seen" and "Retired" for the status options.